packages/graphql-yoga/src/server.ts

```diff
@@ -155,6 +164,14 @@ export type YogaServerOptions<
   fetchAPI?: FetchAPI
   multipart?: boolean
   id?: string
+  /**
+   * You can limit the number of batched operations per request.
+   *
+   * Set this to undefined or 0 to disable batching.
+   *
+   * @default 0
+   */
+  batchingLimit?: number
```

Comment:
batching should be `opt-in` in my opinion, we can take inspiration from the error masking configuration:

```
batching: boolean | { limit?: number }
```

Reply:
It is already opt-in. See https://github.com/dotansimha/graphql-yoga/pull/1662/files#diff-40184c12fde3ff11137997025804dc55fa5a6455d9feb557320a4ab37c0c1d10R113
I didn't want to allow `true` because `true` would mean `Infinity` but instead I encourage people to give an exact number for the limit

Reply:
One argument from my side on why we should use `batching: boolean | { limit?: number }` instead of just a `batchingLimit` is that we could potentially group future `batching` options under the `batching` key without introducing a breaking change release.

Right now I have no examples for additional options, but past experience showed that this might happen eventually. As for `batching: true, the default config could be something like `{ limit: 10 }`.